rules/utils.bzl

```diff
@@ -0,0 +1,4 @@
+"""Common helper macros that could be used in different BUILD and/or extension files"""
+
+def bundle_identifier_for_bundle(bundle_name):
+    return "com.cocoapods." + bundle_name
```

Comment:
CocoaPods applies some validation on this value, a lot of that logic requires regex support so we won't be able to reach parity here easily. This value was already being set before to what you see here, so don't see any harm in continuing to use it for now